stable/cachet/values.yaml

```diff
@@ -24,6 +24,9 @@ ingress:
   host: chart-example.local
   tls: []
   path: /
+  # -- Set ingressClass for Ingress when needed
+  ingressClassName: ingressClass
```

Comment:
I think this should be commented by default otherwise `ingressClass` will be used?

```suggestion
  # ingressClassName: ingressClass
```

Reply:
yes, you are right. updated it.